addons/mail/static/src/attachments/attachment_viewer.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.AttachmentViewer" owl="1">
         <div class="d-flex justify-content-center" t-att-class="{ 'modal fixed-top bottom-0': props.modal }">
             <div class="o-mail-AttachmentViewer flex-column align-items-center d-flex w-100 h-100" tabindex="0" t-ref="autofocus" t-on-keydown.stop="(ev) => this.onKeydown(ev)">
-                <div class="o-mail-AttachmentViewer-header position-absolute top-0 d-flex w-100 bg-black-75 text-400">
+                <div class="o-mail-AttachmentViewer-header position-absolute top-0 d-flex w-100 bg-black-75 text-400" t-on-click.stop="">
```

Comment:
can you add the link to the enterprise PR (and same on the enterprise PR) and the task id ?